lib/shared/addon/components/container/form-volumes/component.js

```diff
@@ -11,12 +12,17 @@ export const EXISTING_PVC = 'existingPvc';
 export const BIND_MOUNT = 'bindMount';
 export const TMPFS = 'tmpfs';
 export const SECRET = 'secret';
+export const CUSTOM_LOG_PATH = 'customLogPath';
+
+export const LOCAL_FLEXVOLUME = 'rancher.io/localflexvolume'
```

Comment:
Did this get renamed?

Reply:
No, I added that.

Reply:
I need to distinguish local flex volume from flex volume according to the `volume.flexvolume.options.dirver` field which either  `"rancher.io/localflexvolume"` or `"rancher.io/flexvolume"` .

Reply:
I mean I thought we were renaming the flex driver to `log-aggregator` or something, because "localflexvolume" conveys no meaning about what it does.